recipes/ade/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class AdeConan(ConanFile):
+    name = "ade"
+    license = "Apache-2.0"
+    homepage = "https://github.com/opencv/ade"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Graph construction, manipulation, and processing framework"
+    topics = ("graphs", "opencv")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version][0],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder,
+                        build_folder=self._build_subfolder)
+        cmake.build()
+        self._cmake = cmake
+
+    def package(self):
+        self._cmake.install()
```

Comment:
Please move the configure to a `_configure_cmake` method and call it in `build` and `package`.
The current state does not support `conan create . ade/0.1.1f@ -kb`